src/Servers/Kestrel/Core/src/Internal/Http/Http1Connection.cs

```diff
@@ -620,6 +622,21 @@ protected override void BeginRequestProcessing()
             TimeoutControl.SetTimeout(_keepAliveTicks, TimeoutReason.KeepAlive);
         }
 
+        protected override async Task ProcessRequestAsync<TContext>(IHttpApplication<TContext> application, TContext context)
```

Comment:
ValueTask?

Reply:
Yea, that'll at least remove the allocation